File: common/src/main/java/org/sandboxpowered/silica/network/play/clientbound/EntityStatus.java
package org.sandboxpowered.silica.network.play.clientbound;

import org.sandboxpowered.silica.network.PacketByteBuf;
import org.sandboxpowered.silica.network.PacketHandler;
import org.sandboxpowered.silica.network.PacketPlay;
import org.sandboxpowered.silica.network.PlayConnection;

public class EntityStatus implements PacketPlay {
    @Override
    public void read(PacketByteBuf buf) {

    }

    @Override
    public void write(PacketByteBuf buf) {
        buf.writeInt(0);
        buf.writeByte(24);
    }

    @Override
    public void handle(PacketHandler packetHandler, PlayConnection connection) {

    }
}

File: common/src/main/kotlin/org/sandboxpowered/silica/network/play/clientbound/EntityStatus.kt
package org.sandboxpowered.silica.network.play.clientbound

import org.sandboxpowered.silica.network.PacketByteBuf
import org.sandboxpowered.silica.network.PacketHandler
import org.sandboxpowered.silica.network.PacketPlay
import org.sandboxpowered.silica.network.PlayContext

class EntityStatus : PacketPlay {
    override fun read(buf: PacketByteBuf) {}
    override fun write(buf: PacketByteBuf) {
        buf.writeInt(0)
        buf.writeByte(24)
    }

    override fun handle(packetHandler: PacketHandler, context: PlayContext) {}
}